openhands/controller/agent_controller.py

```diff
@@ -58,6 +58,7 @@
 TRAFFIC_CONTROL_REMINDER = (
     "Please click on resume button if you'd like to continue, or start a new task."
 )
+ERROR_ACTION_NOT_EXECUTED = 'The action has not been executed. This likely occurred because the runtime system crashed due to resource limitations. Note that the OpenHands agent has limited resources available for running commands. The agent can retry this action - the runtime system can be rebooted when retrying. However, be aware that any dependencies, environment variables, or other system state previously set up by the agent might be lost after the reboot process.'
```

Comment:
```suggestion
ERROR_ACTION_NOT_EXECUTED = 'The action has not been executed. This likely occurred because the runtime system crashed and was rebooted due to resource limitations. Be aware that any dependencies, environment variables, or other system state previously set up by the agent may have been lost after the reboot process.'
```